coop_memberspace/views/website/website_template.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+    <!-- Invisible Menu if user don't login -->
+        <!-- <template id="coop_memberspace.layout" inherit_id="website.layout" name="Main layout">
+            <xpath expr="//ul[@id='top_menu']/t" position="replace">
+                <t t-foreach="website.menu_id.child_id" t-as="submenu">
+                    <t t-if="submenu and website.user_id != user_id">
+                        <t t-call="website.submenu" />
+                    </t>
+                </t>
+            </xpath>
+
+            <xpath expr="//div[@class='navbar navbar-default navbar-static-top']//ul[@class='dropdown-menu js_usermenu']/li[@id='o_logout']" position="before">
+                <li ><a href="/profile" role="menuitem">Profil</a></li>
+            </xpath>
+
+            <xpath expr="//div[@class='navbar navbar-default navbar-static-top']//ul[@class='dropdown-menu js_usermenu']/li[@id='o_logout']/a" position="replace">
+                <a t-attf-href="/web/session/logout?redirect=/" role="menuitem">Se déconnecter</a>
+            </xpath>
+        </template>
+
+        <template id="coop_memberspace.show_sign_in" customize_show="True" inherit_id="website.show_sign_in" name="Show Sign In">
+            <xpath expr="//a[@t-attf-href='/web/login']" position="replace">
+                <a t-attf-href="/web/login">
+                    <b>Se connecter</b>
+                </a>
+            </xpath>
+        </template> -->
```

Comment:
Did you commented all of these?
If there's no use for them just remove them.

But can you please explain it?